install/includes/database_structure.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+return [
+    'perm_items' => [
+        'columns' => [
+            'id' => ['type' => 'integer', 'options' => ['autoincrement' => true ]],
+            'name' => ['type' => 'string'],
```

Comment:
what will be the length of this field? Same goes to other fields which type is string.